libraries/plugins/json_rpc/json_rpc_plugin.cpp

```diff
@@ -192,7 +195,8 @@ namespace detail
    {
       FC_UNUSED( lock )
       FC_UNUSED( notify_callback )
-      return _methods;
+      string network_name = args ? *args : _default_network;
+      return _methods[network_name];
```

Comment:
this will create default empty vector in _methods in case args does not contain existing network_name, better to use at() so it throws exception ?

Reply:
Right, at() or check for existence is better at this place, will rework